redbot/cogs/trivia/data/lists/startrek.yaml

```
@@ -0,0 +1,3159 @@
+AUTHOR: Scarecr0w12
+In What Year Did The Vulcans Make Official First Contact With Zefram Cochrane?:
+- 2063
+In What State Is Carbon Creek Located?:
+- pennsylvania
+Before T?pol What Was The Longest A Vulcan Had Served Aboard A Human Ship?:
+- ten Days
+What Are T?pol, Trip And Archer Drinking?:
+- wine
+What Did The Vulcan Vessel Go To Earth In 1957 To Investigate?:
+- sputnik
+What Type Of Game Did Maggie Take Mestral To See?:
+- baseball
+Where Did The Vulcans Make Official First Contact With Zefram Cochrane?:
+- bozeman, Montana
+What Was Mestral Watching On The Tv In The Bar?:
+- atomic Bomb Test
+What Species Forwarded The Distress Signal To The Vulcan High Command?:
+- tellarite
+What Is A Vulcan Second Foremother?:
+- a Mother?s Mother?s Mother
```

Looks like a `'` got turned into a `?` here.
```suggestion
- a Mother's Mother's Mother
```

---

Totally can, Kinda forgot about this over time, but i can finish this out, no problem.

---

> Totally can, Kinda forgot about this over time, but i can finish this out, no problem.

Hi there, I don't want this to sound like I'm pushing you but if you don't have the time for this I can get it wrapped up and approved, just let me know and I'll be happy to reformat and help. Please let us know soonish if you can, this item is slated to hopefully be included with the next update if everything goes well. Again sorry for being a bit pushy about time when this has been sitting for so long.